components/tools/OmeroPy/test/integration/test_search.py

```diff
@@ -117,9 +117,7 @@ def _3164_setup(self, owner):
 
         images = list()
         for i in range(0, 5):
-            img = omero.model.ImageI()
-            img.name = omero.rtypes.rstring("search_test_%i.tif" % i)
-            img.acquisitionDate = omero.rtypes.rtime(0)
+            img = self.new_image(name="search_test_%i.tif" % i)
             tag = omero.model.TagAnnotationI()
             tag.textValue = omero.rtypes.rstring("tag %i" % i)
```

Comment:
Is there any worth in tweaking the  new tag helper here or is this a one-off use?